Fix:
drivers/rotator/deepskydad_fr1.cpp:280:9: error: call to 'abs' is ambiguous
https://github.com/indilib/indi/issues/1608
--- drivers/rotator/deepskydad_fr1.cpp.orig	2022-01-17 04:21:47.000000000 -0600
+++ drivers/rotator/deepskydad_fr1.cpp	2022-01-30 04:37:14.000000000 -0600
@@ -27,6 +27,7 @@
 #include "connectionplugins/connectionserial.h"
 
 #include <cerrno>
+#include <cmath>
 #include <cstring>
 #include <memory>
 #include <termios.h>
